src/adaptors/http1/http1_adaptor.c

```diff
@@ -342,17 +390,28 @@ void qdr_http1_free_written_buffers(qdr_http1_connection_t *hconn)
 // Raw Connection Read Buffer Management
 //
 
-int qdr_http1_grant_read_buffers(qdr_http1_connection_t *hconn)
+static size_t _raw_connection_grant_read_buffers(pn_raw_connection_t *pn_raw_conn)
```

Comment:
can this function be replaced with a call to https://github.com/skupperproject/skupper-router/blob/main/src/adaptors/adaptor_common.c#L233  ?

Reply:
Done